src/injected/ui.cpp

```diff
@@ -383,7 +383,7 @@ void hook_savegame()
     {
         register_on_write_to_file([](const char* backup_file, const char* file, void* data, size_t data_size, WriteToFileOrig* original)
                                   {
-                                      if (file == "savegame.sav"sv and options["disable_savegame"])
+                                      if (strcmp(file, "savegame.sav") == 0 and options["disable_savegame"])

```

Comment:
why tho?

Reply:
`file` is `const char*`, so before it was comparing pointers not strings

Reply:
What is was doing before was comparing a string_view to a const char*, which is perfectly well defined behavior

Reply:
oh, is that way better than `strcmp`? 
cuz this is the only use of `string_view` in the entire UI

Reply:
It's not better or worse, just more C++ as opposed to using the C stl
I mainly thought the diff was unnecessary.